tests/fv3_conf/fv3_qsub.IN_cheyenne

```diff
@@ -13,7 +13,9 @@ echo -n " $( date +%s )," >  job_timestamp.txt
 set +x
 MACHINE_ID=cheyenne
 source ./module-setup.sh
-module use $( pwd -P )
+mkdir makefiles
```

Comment:
Directory should be named `modulefiles` not `makefiles`

Reply:
oops! fixed it